src/style.css

```diff
@@ -1,3 +1,76 @@
-body {
+@font-face {
+  font-family: Roboto;
+  src: url('./fonts/Roboto-Medium.ttf') format('truetype');
+  font-weight: 500;
+  font-style: normal;
+}
+
+* {
   margin: 0;
+  list-style: none;
+  text-decoration: none;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.logo {
+  vertical-align: middle;
+}
+
+.menu__list {
+  display: flex;
+  align-items: center;
+  padding: 0;
+}
+
+.menu__link {
+  display: flex;
+  align-items: center;
+  height: 60px;
+  color: #000;
+}
+
+.menu__list .menu__item:nth-child(4) .menu__link:hover {
+  color: rgb(0, 172, 220);
+}
+
+.menu__item {
+  font-size: 12px;
+  text-transform: uppercase;
+  margin-right: 20px;
+}
+
+.menu__item :first-child {
```

Comment:
The selector `.menu__item :first-child` has a space before `:first-child`, which may cause it to select the first child of any descendant of `.menu__item`. If the intention is to select the first `.menu__item`, use `.menu__item:first-child` without the space.